executor/new_distsql.go

```diff
@@ -50,12 +50,17 @@ type TableReaderExecutor struct {
 	dagPB     *tipb.DAGRequest
 	ctx       context.Context
 	schema    *expression.Schema
-	// columns are only required by union scan.
-	columns []*model.ColumnInfo
+	columns   []*model.ColumnInfo
 
 	// result returns one or more distsql.PartialResult and each PartialResult is returned by one region.
 	result        distsql.SelectResult
 	partialResult distsql.PartialResult
+
+	// aggregate indicates whether it contains Aggregation executors or not.
+	aggregate bool
```

Comment:
If the aggregation is pushed down to storage engine, it will affect the behavior or generated columns.